catala.opam

```diff
@@ -32,6 +32,7 @@ depends: [
   "js_of_ocaml-ppx" {>= "3.8.0"}
   "camomile" {>= "1.0.2"}
   "odoc" {with-doc}
+  "cppo" {>= "1"}
```

Comment:
The `catala.opam` is generated from `dune-project`, so you should modify this one. There's also a list of `opam` packages to fetch in the `dependencies-ocaml` rule of the top-level `Makefile`.

Reply:
Ah yes, just noticed this about the dune file, sorry !

Also, I have patches to avoid the need for listing the dependencies again in the Make file (using `opam install . --deps-only` instead) but haven't cleaned up for pushing yet :)